.clang-format

```diff
@@ -81,7 +81,7 @@ FixNamespaceComments: true
 IndentWrappedFunctionNames: false
 IncludeBlocks:   Regroup
 IncludeCategories:
-  - Regex:           '^<cuda/experimental/__async/sender/prologue.cuh>'
+  - Regex:           '^<cuda/experimental/__execution/prologue.cuh>'
```

Comment:
We should not include those files into libc++

If there are issues we should solve either globally, or locally. Also as currently written the prologue would suppresses warnings without reenabling them which is a nogo

Reply:
I cannot attach a comment to the file, but the same applies for the clang-format file.

I am strongly opposed to subfolder formatting rules. Rather than that we should see whether we can include that option into our global format file